configure-fuzzing.sh

```diff
@@ -0,0 +1,374 @@
+#!/bin/bash
```

Comment:
I think the configuration file can be much simpler in this case. No need to have all of these command-line parameter etc.

It can be pretty basic and just add the necessary flags to compile PcapPlusPlus with fuzzing. What do you think?

Reply:
I would keep all the flags that the other `configure-*.sh` files have, because this first fuzzer might not need it but in the future it might be good to implement new fuzzers to get more coverage on all the library code base.

Reply:
I usually try to keep things simple in PcapPlusPlus. My suggestion would be to keep the minimal options required for fuzzing and add options later as needed

Reply:
As you wish then. I've stripped down the `configure-fuzzing.sh` file to have the minimal options. It should be ready for merge now.